### PR TITLE
feat: added postcss-unused-var to default preset

### PR DESCRIPTION
**Pull request by anikethsaha**
Added `postcss-unused-var` to default preset

Docs : https://github.com/anikethsaha/postcss-unused-var#readme

Updated the preset docs.

### CODE REVIEW

**Comment by anikethsaha** [comment]
@evilebottnawi 

### 1. 
image

I think this test should be updated. If we are removing the un-used `var` then this test needs to be updated. Removing the whole unused var is the correct minification for `postcss-unused-var`

### 2.

image

Its `mixins` so it has been handled in https://github.com/cssnano/cssnano/pull/857/commits/45edd58a438b194ec5d3e6a02716d71c8d943173 as mentioned here https://github.com/cssnano/cssnano/pull/857#issuecomment-565983142

**Comment by anikethsaha** [comment]
> Maybe better move code to this repo? For fast fixes and features

Yea sure,

But still we need to update those tests

**Comment by evilebottnawi** [comment]
Feel free to do it

**Comment by anikethsaha** [comment]
@evilebottnawi tests are working fine in local machine.
Anything missing ?

**Comment by anikethsaha** [comment]
image

@evilebottnawi I don't know why this error is occurring. I updated the babel alias, update the preset still it's failing.

any help !!!

**Comment by anikethsaha** [comment]
image

@evilebottnawi tests are passing in local machine

**Comment by anikethsaha** [comment]
cc @evilebottnawi all tests are passing.

File: packages/cssnano-preset-default/package.json [hunk 1]
```diff
@@ -43,7 +43,8 @@
     "postcss-reduce-initial": "^4.0.3",
     "postcss-reduce-transforms": "^4.0.2",
     "postcss-svgo": "^4.0.2",
-    "postcss-unique-selectors": "^4.0.1"
+    "postcss-unique-selectors": "^4.0.1",
+    "postcss-unused-var": "^0.1.1"
```
Comment (evilebottnawi) [inline packages/cssnano-preset-default/package.json]: Please start plugin with `1.0.0` version

Comment (anikethsaha) [reply packages/cssnano-preset-default/package.json]: ok I will publish v1 first and then I will change it here 👍

**Comment by anikethsaha** [comment]
yes...thats why this plugin is not removing variables which are declared in `:root`

https://github.com/anikethsaha/postcss-unused-var#output

**Comment by evilebottnawi** [comment]
Still unsafe:

**index.html**

```css
<head>
  <link rel="stylesheet" href="./styles.css">
  <link rel="stylesheet" href="./styles-other.css">
</head>
<div>Test</div>
```

**style.css*

```css
:root {
  --root-var-color: red;
}
div {
  --main-txt-color: blue;
  --main-padding: 15px;
  --main-bg-color: coral;
  background-color: var(--main-bg-color);
}
```

**styles-other.css**

```css
div {
  color: var(--main-txt-color);
  padding: var(--main-padding);
}
```

**Pull request closed without merging**